static/styles/subscriptions.css

```diff
@@ -200,10 +200,10 @@
     color: hsl(0, 100%, 50%);
 }
 
-.sub_settings_title {
+.stream_setting_subsection_title {
+    font-size: 19.5px;
```

Comment:
Where does the 19.5px value come from?

Reply:
I changed it to `1.5em` which is what we use in headings in "Settings".

(BTW originally I had copied it from settings but later got changed to `19.5px` based on trying out different sizes to check what suits most with other text and headings around.)